tutorials-dp0-3/portal-dp0-3-3.rst

```diff
@@ -0,0 +1,425 @@
+You can exclude those objects from your analysis by entering ``< 1`` in the box underneath the table heading ``e``, and hitting the carriage return.  
+To see the distribution of ``q`` vs. ``e`` more clearly, also restrict the range of ``q`` by entering ``< 50`` and hitting the carriage return.  
+This will result in a slightly modified display as below.  
+
+.. figure:: /_static/portal_tut03_step01b.png
+    :width: 600
+    :name: portal_tut03_step01b
+    :alt: A screenshot of the default results view for the modified query.
+
+    The view for the query with e < 1 and q < 50.  
+
+
+1.6. Create a column of semi-major axis, ``a``.
+In the upper right column of the table panel, click on the icon to add a column (a tall narrow rectangle to the left of a + sign).
+In the pop-up window to "Add a column", set the "Name" to "a", the "Expression" to "q/(1-e)", the "Units" to "au",
+and the "Description" to "semi-major axis".  
+Click "Add Column", and see the new column appear in the table.
+
+.. figure:: /_static/portal_tut03_step01c.png
+    :width: 400
+    :name: portal_tut03_step01c
+    :alt: A screenshot of the pop-up window to add a column.
+
+    The "Add a column" pop-up window.  
+
+
+1.7. Create a scatter plot of inclination vs. semi-major axis.
+In the plot panel, click the "Settings" icon (double gears), and select "Add New Chart".
+Set the "Plot Type" to "Scatter", the "X" to "a", "Y" to "incl".
+Set the "X Min" to "0", the "X Max" to 60, the "Y Min" to 0, and the "Y Max" to 80.
+Set the axis labels as shown in the figure below.
+Click "OK".
+
+.. figure:: /_static/portal_tut03_step01d.png
+    :width: 400
+    :name: portal_tut03_step01d
+    :alt: A screenshot of the plot parameters pop-up window.
+
+    Create a new plot with these parameters.
+
+
+1.8. Delete the default plot by clicking on the blue cross in the upper right corner, so that only the newly-created plot appears (it should look like the plot below).
+TNOs appear as a distinct population with ``a`` > 30.1 AU in this parameter space.
+
+.. figure:: /_static/portal_tut03_step01e.png
+    :width: 600
+    :name: portal_tut03_step01e
+    :alt: A screenshot of the inclination versus semi-major axis plot, showing a clear population of TNOs.
+
+    The population of TNOs has x-values greater than 30 au.
+
+
+1.9.  Now that you've identified the population of the Trans-Neptunian Objects, you can further study their properties.  
+To begin with, we will plot the eccentricity ``e`` vs. semi-major axis ``a`` of the orbit.  
+You can re-execute the query from Step 1.2, but now include in your query all objects with ``a > 30.1``, with bound orbits (``e < 1``) and those with more than 10 observations 
+(note no restriction ``WHERE ssObjectId > 9000000000000000000`` which we used in Section 1.2).  
+Note that requring more than 10 observations, you will have to perform a table join on the ``dp03_catalogs_10yr.SSObject`` table as below.  
```

Comment:
When a new query is to be made, include instructions for how to navigate back to the search page.

Reply:
OK